mana_mushlemet.py

```diff
@@ -0,0 +1,34 @@
+
+
+def is_complete(num_pieces: int) -> bool:
+    """
+    The function get a number and check if it complete meal
+    :param num_pieces: number of pieces in the meal
+    :return: true if the meal is complete and false if not
+    """
+    divisions = [div for div in range(1, num_pieces) if num_pieces % div == 0]  # Finding the possible divisions
+    return num_pieces == sum(divisions)  # check if the number == the sum of it's possible divisions
+
+
+def complete_meals() -> int:
+    """
+    The function returns the complete numbers
```

Comment:
I think a better term would be "perfect number".
Also, please explain what is a complete number / perfect number.